CMakeLists.txt

```diff
@@ -759,7 +764,7 @@ if(NOT ANDROID)
       message(STATUS "Using static hidapi from Externals")
       add_subdirectory(Externals/hidapi/mac)
       list(APPEND LIBS hidapi)
-    elseif(${CMAKE_SYSTEM_NAME} STREQUAL "Linux")
+    elseif((${CMAKE_SYSTEM_NAME} STREQUAL "Linux") AND HAVE_UDEV)
```

Comment:
Would the test be correct if hidapi-libusb could compile with the libusb from Externals?